ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -87,3 +97,96 @@ - (UIView *)view {
 }
 
 @end
+
+@implementation CustomButtonAnnotationToolbar
```

Comment:
Could we move this class to a new file to not clutter up the main implementation?

Reply:
Since this PR will never be merged to master and since its sole purpose is to serve as an example, we want to avoid using separate files, as adding new classes in Xcode is more complicated than just adding new code in an existing file. We want to make it easy to use for develeopers who are not familiar with Xcode.